tests/snmpsim/querx.snmprec

```diff
@@ -0,0 +1,2 @@
+1.3.6.1.2.1.1.1.0|4|Querx sensor
+1.3.6.1.2.1.1.2.0|6|.1.3.6.1.4.1.3444.1.14
```

Comment:
This test data is not complete (no sensors included). Please regenerate this snmprec file.